alerts/cluster-network-operator/NorthboundStaleAlert

```diff
@@ -0,0 +1,22 @@
+# Northbound Stale Alert
+
+## Meaning
+
+The alert `NorthboundStale` is triggered when ovn-northd has gone longer than 2 minutes 
+without succesfully synching any changes to the Northbound Database.
+
+## Impact 
+
+Networking control plane is degraded and Networking Configuration updates applied to the cluster will not be applied.
+Existing workloads should continue to have connectivity, but the OVN-Kubernetes control plane and/or the OVN southbound database
+may not be functional. 
+
+## Diagnosis
+
+Either ovn is overloaded, i.e. the load on the cluster is too large due to a large number of services, network policies, pods, or nodes, or the control plane
```

Comment:
You must break the diagnosis down into 3 areas.
* Investigate ovnkube-master process and ensure its functioning i.e. check that the container is running. If it is not, check the logs and resolve any issues there. Take special care to see if there is an error for updating NB DB timestamp. We dont want to doc what this error message will be exactly because it could change so keep it generic. Please give all the oc commands to do all of this. 

* Investigate whether OVN NB DB is functioning. Ensure the NB DB containers are running without an error. If there is an error, triage that first. If there is no error, ensure there is one OVN NB DB leader. Give the commands to check for at least one OVN NB DB leader. Attempt to connect to the NB DB leader using ovn-nbctl show or whatever just to check the NB DB leader is accepting connections.

* If all else above is working ok, then check if connectivity between ovnkube-master leader and OVN NB DB cluster leader is ok.